package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "author": "Fritz Lekschas (https://lekschas.de)",
   "license": "MIT",
-  "homepage": "https://higlass.io",
+  "homepage": "http://localhost:3000/hg/",
```

Comment:
Shouldn't this be `.` to support relative paths? Either way, I guess `http://localhost:3000/hg/` was just accidentally added? :)

Reply:
Sorry, that was just for testing. It should be removed.